modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,97 @@
+// Used in cicd/builds/running-entitled-builds.adoc
+
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running entitled builds with SharedSecret objects
+
+[role="_abstract"]
+To avoid copying your {op-system-base} subscription credentials and certificates into the builds' namespaces, you can install entitled RPM packages during builds. To install entitled RPMs, you run entitled builds with `SharedSecret` objects that import {op-system-base} Simple Content Access (SCA) certificates.
+
+This procedure relies on the newly-introduced Shared Resources feature and the Insights Operator in {product-title}.
+
+[IMPORTANT]
+====
+The Shared Resources feature is only available as a Technology Preview feature. It also belongs to the `TechPreviewNoUpgrade` feature set, which are not part of the default features, and are not recommended on production clusters. To use the Shared Resources feature, you must enable the `TechPreviewNoUpgrade` feature set. Once enabled, you cannot disable the `TechPreviewNoUpgrade` feature set and cannot upgrade your cluster. See xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-tp-tasks[Enabling Technology Preview features using FeatureGates].
+====
+
+.Prerequisites
+
+* You have xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-tp-tasks[enabled the  `TechPreviewNoUpgrade` feature set by using the FeatureGates].
+
+.Procedure
+
+Under the [examples directory](examples) there is both a simple example of using a `SharedConfigMap` in a `Pod`, and an
```

Comment:
Maybe you already had this planned since this is a WIP, but it not, sorry @rolfedh but the entire Procedure section here needs to be changed to *NOT* use direct references to the examples dir in our source code repo.

I *think* you are planning to borro from the blow post draft, right?

If so, yes, a filtered down version of the "What can we demonstrate" section https://docs.google.com/document/d/1SACY6vX0IaxIUr4cL6g1tqLDVM7Oh3NHi5DbcwIu8Vk/edit#heading=h.220lhy819rda of the blog post is what is needed.

Do you feel comfortable doing that filtering / pruning / paring down, or do you need me to take a stab at it?

Reply:
This is excellent feedback, @gabemontero. I appreciate it. I'll be moving this content over today so that you can review it early on Monday. I'll incorporate your changes and try to get this PR merged by EOD Monday.

Reply:
Updated. Thanks. Ready for your review.